content/docs/searching.md

```diff
@@ -1339,23 +1339,26 @@ Next will be a **1702** message that contains a bit-weighted number in **lParam*
 
 `<Action type="3" message="1702" wParam="0" lParam="515" sParam="" />`
 
-| 1702-Bit-Weight |Binary-Bit-Weight  | Meaning (equivalent option checked) |
-|----------------:|------------------:|:-----------------------------------|
-| 1               | 0000000001        | Match whole word only              |
-| 2               | 0000000010        | Match case                         |
-| 4               | 0000000100        | Purge for each search              |
-| 16              | 0000010000        | Bookmark line                      |
-| 32              | 0000100000        | In all sub-folders                 |
-| 64              | 0001000000        | In hidden folders                  |
-| 128             | 0010000000        | In selection                       |
-| 256             | 0100000000        | Wrap around                        |
-| 512             | 1000000000        | Backward direction (*)             |
-
-*: **Backward direction** checked means 512 is _not_ included; unchecked means 512 _is_ included.
+| 1702-Bit-Weight |Binary-Bit-Weight      | Meaning (equivalent option checked) | Alternate Meaning (\*\*) |
+|----------------:|----------------------:|:------------------------------------|:-----------------------|
+| 1               | 0000-0000-0001        | Match whole word only               |                        |
+| 2               | 0000-0000-0010        | Match case                          |                        |
+| 4               | 0000-0000-0100        | Purge for each search               |                        |
+| 16              | 0000-0001-0000        | Bookmark line                       |                        |
+| 32              | 0000-0010-0000        | In all sub-folders                  |                        |
+| 64              | 0000-0100-0000        | In hidden folders                   |                        |
+| 128             | 0000-1000-0000        | In selection                        | Project Panel 1 (\*\*)   |
+| 256             | 0001-0000-0000        | Wrap around                         | Project Panel 2 (\*\*)   |
+| 512             | 0010-0000-0000        | Backward direction (\*)              | Project Panel 3 (\*\*)    |
+| 1024            | 0100-0000-0000        | . matches newline                   |                        |
+
+\*: **Backward direction** checked means 512 is _not_ included; unchecked means 512 _is_ included.
+
+\*\*: **Project Panel "alternate meaning"** column shows the meaning for those those bits when the action **1701** message (below) is set to **Find All** (in Projects) or **Replace in Projects**.
```

Comment:
Not sure backslashes are truly needed here?

Reply:
Hmm, maybe they are, but curious they don't appear in the pink part of the diff.

Reply:
Needed?  No.  But they help lexing tools (specifically, the Markdown UDL) disambiguate.  And they even help _my_ eyes disambiguate, since I interpret any first-character-as-asterisk as being a probable bullet-list item, and secondarily being a bold-or-italic starter.  

Actually, I just talked myself into using ¹ and ² rather than * and ** to avoid any confusion.